MedallionShell/Streams/MergedLinesEnumerable.cs

```diff
@@ -20,6 +26,18 @@ public MergedLinesEnumerable(TextReader standardOutput, TextReader standardError
             this.standardError = standardError;
         }
 
+#if NETSTANDARD2_0_OR_GREATER
+        public IAsyncEnumerator<string> GetAsyncEnumerator(CancellationToken cancellationToken = default)
+        {
+            Throw<InvalidOperationException>.If(
```

Comment:
This line is now duplicated. Let’s share a private function AssertNoMultipleEnumeration